manual/Tasks/symlink.html

```diff
@@ -26,7 +26,7 @@
 
 <h2><a name="symlink">Symlink</a></h2>
 <h3>Description</h3>
-<p> Manages symbolic links on Unix based platforms. Can be used to
+<p> Manages symbolic links on platforms that support symbolic links. Can be used to
```

Comment:
rather "platform where Java supports symbolic links"? I'm sure people will yell at us as their platform supports some kind of symbolic link but Java doesn't support it.

Reply:
Fixed and updated the PR